frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "i18next": "^10.5.0",
     "js-cookie": "^2.2.0",
     "lodash": "^4.17.11",
+    "lodash.uniqby": "^4.7.0",
```

Comment:
I forgot that lodash was already a dependency
so remove this line (using the npm command line)